src/client/public/styles/dash.css

```diff
@@ -7,38 +7,55 @@ a {
   color: inherit;
 }
 
-.pr-list h2 {
-  grid-column: 1 / -1;
-}
-
 .pr-list {
   max-width: var(--max-desktop-width);
   margin: 0 auto;
 
-  display: grid;
-  align-items: center;
-  grid-template-columns: auto 40px 1fr;
-  grid-gap: 0px var(--padding);
-
   font-size: 14px;
 }
 
-.pr-author {
-  grid-column: 1 / 1;
+.pr {
+  position: relative;
+  --avatar-size: 40px;
```

Comment:
I would put this into the `dimens.css` file. Rationale being that if there are common patterns for the UI for follow - it's clearer if the values are in one spot - it also encourages re-use, which may help with future refactors.

Reply:
Done.